src/milia/api/user.cljc

```diff
@@ -43,12 +43,12 @@
 
 (defn get-subscription
   [username]
-  (let [url (make-url "pricing" "subscription" "zoho" username)]
+  (let [url (make-url "pricing" "subscriptions" username)]
     (parse-http :get url :suppress-4xx-exceptions? true)))
 
 (defn get-subscription-payment
```

Comment:
Could these functions names `get-subscription-payment` and `get-subscription` be renamed to `get-subscriptions-payments` and `get-subscriptions` then. No?

Reply:
May be not because each of them is retrieving a single instance of whatever it's retrieving. The `s` that was added was mainly because the urls in pricing have been updated.